service/cloudconfigv1/v_1_0_0_master_template.go

```diff
@@ -3,18 +3,18 @@ package cloudconfigv1
 import (
 	"github.com/giantswarm/certificatetpr"
 	clustertprspec "github.com/giantswarm/clustertpr/spec"
-	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_0_1_0"
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_1_0_0"
 	"github.com/giantswarm/kvmtpr"
 	"github.com/giantswarm/microerror"
 )
 
-func v_0_1_0MasterTemplate(customObject kvmtpr.CustomObject, certs certificatetpr.AssetsBundle, node clustertprspec.Node) (string, error) {
+func v_1_0_0MasterTemplate(customObject kvmtpr.CustomObject, certs certificatetpr.AssetsBundle, node clustertprspec.Node) (string, error) {
```

Comment:
I also did the renaming now because the naming reflects the import and the supported version.